pkg/test/framework/components/istio/config.go

```diff
@@ -213,6 +213,10 @@ type Config struct {
 	// ControlPlaneInstaller allows installation of custom control planes on istio deployments via an external script
 	// This field should only be set when DeployIstio is false
 	ControlPlaneInstaller string
+
+	// TrustedZtunnelNamespace allows to override the trusted namespace of ztunnel daemonset location.
```

Comment:
```suggestion
	// TrustedZtunnelNamespace lets you override the namespace where the ztunnel daemonset is deployed.
```

Reply:
@sridhargaddam 

Please, fix me if I'm wrong, but "trustedZtunnelNamespace" does not control when ztunnel will be deployed, but rather specify which namespace should be trusted by istiod for the node proxy ztunnels.

https://github.com/istio/istio/blob/master/operator/pkg/apis/values_types.proto#L950-L952

Reply:
Yeah, override could be a bit misleading. How about the following setnence.

```
TrustedZtunnelNamespace lets you specify the namespace where the ztunnel daemonset is deployed.
```